package.json

```diff
@@ -14,5 +19,11 @@
     "mongodb": "^2.2.28",
     "path": "^0.12.7",
     "url": "^0.11.0"
+  },
+  "devDependencies": {
+    "electron": "^1.4.3",
```

Comment:
already in dependencies. remove electron from dependencies, and update the version to 1.6.6 here